src/app/modules/trezor/trezorAccount.js

```diff
@@ -0,0 +1,66 @@
+import * as nemSdk from "nem-sdk";
+import { Observable } from "rxjs";
+import { PublicAccount, NEMLibrary } from "nem-library";
+
+/**
+ * TrezorAccount model
+ */
+export class TrezorAccount {
+ 
+    /**
+     * Constructor
+     * @param address
+     */
+    constructor(address, hdKeyPath, Trezor) {
+        this.address = address;
+        this.hdKeyPath = hdKeyPath;
+        this._Trezor = Trezor;
+    }
+
+    /**
+     * Sign a transaction
+     * @param transaction
+     * @param receiverPublicKey (optional): if given, the message will be encrypted
+     * @returns {{data: any, signature: string}}
+     */
+    signTransaction(transaction, receiverPublicKey) {
+        transaction.signer = PublicAccount.createWithPublicKey("462ee976890916e54fa825d26bdd0235f5eb5b6a143c199ab0ae5ee9328e08ce");
```

Comment:
Copied this class from https://github.com/shierve/nem-trezor/blob/master/src/trezor-account.ts
Not sure why we need this public key. Do you have any ideas @gimre-xymcity @Jaguar0625 ?

Reply:
i don't know, but my guess would be it's placeholder for serialization. try replacing it with zeroed key.

Reply:
I removed all 4 `PublicAccount.createWithPublicKey("462ee976890916e54fa825d26bdd0235f5eb5b6a143c199ab0ae5ee9328e08ce")` and I don't see any issue in voting module for:
- regular wallet
- Ledger wallet
- Trezor wallet

So I think we can remove it.